packages/dataviews/src/style.scss

```diff
@@ -493,8 +493,23 @@
 	margin-left: $grid-unit-10;
 }
 
-.dataviews-view-grid__card.has-no-pointer-events * {
-	pointer-events: none;
+.dataviews-view-grid__card {
+	position: relative;
+
+	&.has-no-pointer-events {
+		.dataviews-view-grid__content {
+			pointer-events: none;
+		}
+
+		.dataviews-view-grid__overlay {
+			position: absolute;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			cursor: cell;
```

Comment:
If we go with this design, maybe we should change the cursor to a `-` if it's already selected? 🤔

Reply:
I don't think such a cursor exists.

Reply:
The `cell` cursor is only incidentally a ➕, and as @jameskoster says, there is no ➖ equivalent unfortunately.